edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -97,12 +97,12 @@ fi
 
 # Create an edgeagentuser and add it to iotedge group
 if ! /usr/bin/getent passwd %{iotedge_agent_user} >/dev/null; then
-    %{_sbindir}/useradd -r -g %{iotedge_group} -c "edgeAgent user" -s /bin/sh -u %{iotedge_agent_uid} %{iotedge_agent_user} || true
+    %{_sbindir}/useradd -r -g %{iotedge_group} -c "edgeAgent user" -s /bin/nologin -u %{iotedge_agent_uid} %{iotedge_agent_user} || true
```

Comment:
Are you sure you have the right path? A CentOS 7 container has `/sbin/nologin`, not `/bin/login`.

Reply:
Yeah, I think so. I used what's in the file i.e.
https://github.com/Azure/iotedge/blob/release/1.4/edgelet/contrib/centos/aziot-edge.spec#L85

Reply:
That file is also wrong then.

Reply:
Oh ok. I'll use the `sbin`.